help/en/html/setup/index.shtml

```diff
@@ -67,15 +67,31 @@
 
       <ul>
         <li>Version 2.12 can run on any computer system that will
-        run Java 1.5 or later.</li>
+        run Java 1.5 through Java 8 (also known as Java 1.8).</li>
 
-        <li>Version 2.14.1 requires Java 1.5 (or 1.6 if you wish
-        for drag &amp; drop)</li>
+        <li>Version 2.14.1 requires Java 1.5 (or Java 1.6 to support
+        for drag &amp; drop) through Java 8.</li>
 
-        <li>Version 3.10.1 requires Java 1.7 or later.</li>
+        <li>Version 3.10.1 requires Java 1.7 through Java 8.</li>
 
-        <li>Version 4.2 requires Java 1.8.</li>
-      </ul><a name="startprog" id="startprog"></a>
+        <li>Version 4.2 requires Java 8.</li>
+      </ul>
+      
+      <p>As of JMRI 4.20, JMRI is only fully supported when running on the most
+          current update possible to the two most recent Java Long Term Support
+          (LTS) releases (8 or 11 as of July 2020) and will receive "best effort"
+          support on the latest Java release (14 as of July 2020). In this context,
+          "fully supported" means that problems found will be fixed or addressed in
+          other means; "best effort" means that fixes to run on the latest Java
+          release that do not break compatibility with the oldest supported LTS
+          release will be considered.</p>
```

Comment:
I think the Java version discussion is fine.  I'm not sure about saying that fully supported "means that problems found will be fixed or addressed in other means"; we don't always do that.  Could we change that last sentence to focus on something like "If you encounter a problem while using a Java version that is considered 'best-effort' by JMRI, people may not be able to fix any problems you encounter unless you install a fully-supported Java version" or something like that?

Reply:
I've updated the language. Does it work now?